clients/pkg/promtail/targets/journal/journaltarget.go

```diff
@@ -174,12 +174,26 @@ func journalTargetWithReader(
 		return nil, errors.Wrap(err, "parsing journal reader 'max_age' config value")
 	}
 
-	cfg := t.generateJournalConfig(journalConfigBuilder{
+	cb := journalConfigBuilder{
 		JournalPath: targetConfig.Path,
 		Position:    position,
 		MaxAge:      maxAge,
 		EntryFunc:   entryFunc,
-	})
+	}
+
+	matches := strings.Fields(targetConfig.Matches)
```

Comment:
hmm just double checking, if matches are empty will still all journal be checked?

Reply:
Yes. Before PR sdjournal.JournalReaderConfig is created with no matches at all, and all journal is processed.